src/assets/styles/sections/cookie-notification.scss

```diff
@@ -0,0 +1,35 @@
+.cookie-wrapper {
```

Comment:
We're overriding a lot of global styles here. Is that necessary? We shouldn't be setting font size, colors or anything like that at the module level, those should come from higher up.